.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,26 @@
+name: Feature Request
```

Comment:
How about also asking if they'd be willing to implement it?

Reply:
Sure, and we make it a mandatory checkbox :stuck_out_tongue_closed_eyes: